content/reference/templates/index.md

```diff
@@ -1,60 +1,12 @@
-# Templates
+# index.liquid
 
-BookingSync themes are made up of Liquid template files, each serving their own unique purpose. For example, `search.liquid` is used to display multiple rentals, and `rental.liquid` is used to show the details of a single rental.
+The `index.liquid` template renders the homepage of the website. The template is loaded when you go to the root ("/") of the website.
 
-## Theme Structure
+Inside the `index.liquid` template, you may want to show the contents of a specific object. For example, you may want to showcase the Category collection on the homepage.
```

Comment:
`Category collection` is not part of out system, but could be written `list of categories`.